### PR TITLE
Update readmes for 3.0

### LINKED ISSUE #1852: Update all readmes for 3.0 release
### Is your feature request related to a problem? Please describe.

Adjust all Readmes for 3.0 release. Also use this opportunity for general changes where applicable.

### Describe the solution you'd like

Sanity check all Readmes whether are still up to date and that all instructions still makes sense. For example make sure that all quick starts still work.

Also link to the migration guide in general instead of saying that it's just for 1.0 migration.

### Describe alternatives you've considered

No alternatives

### Framework

Core, React, Angular, Vue 3, Vue 2

### RendererSet

Material, Vanilla

### Additional context

Also do it for [JSON Forms 2 Vuetify](https://github.com/eclipsesource/jsonforms-vuetify-renderers).

### PR DESCRIPTION
**Pull request by LukasBoll**
Update readmes for 3.0
added link to migrationguide

closes #1852

Signed-off-by: Lukas Boll lukas-bool@web.de

### CODE REVIEW

File: packages/vue/vue-vanilla/README.md [hunk 1]
```diff
@@ -95,10 +116,25 @@ export default defineComponent({
   },
   data() {
     return {
-      renderers: Object.freeze(vanillaRenderers),
-      data,
-      schema,
-      uischema,
+      renderers: Object.freeze(vanillaRenderers),data: {
```
Comment (sdirix) [inline packages/vue/vue-vanilla/README.md:119]:
```suggestion
      renderers: Object.freeze(vanillaRenderers),
      data: {
```

File: packages/vue/vue/README.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ export default defineComponent({
 ### Renderer Set
 
 The `@jsonforms/vue` package offers JSON Forms Core bindings based on the composition API.
-These bindings handle the props given to the `dispatch-renderer` and use the JSON Forms Core to determine specialized inputs for many use cases like validation and rule-based visibility.
+These bindings handle the props given to the `dispatch-renderer` and use the JSON Forms Core to determine specialized inputs for many use cases like validation and rule-based visibility. This way it´s possible to create custom renderers or entire renderer Sets.
```
Comment (sdirix) [inline packages/vue/vue/README.md:92]:
```suggestion
These bindings handle the props given to the `dispatch-renderer` and use the JSON Forms Core to determine specialized inputs for many use cases like validation and rule-based visibility.
Using these bindings as a basis it's straightforward to create renderer sets for Vue 3.
```

**Review by sdirix — CHANGES_REQUESTED** [review]
Looks good overall. Please take care of the comments ;)

File: packages/vue/vue/README.md [hunk 3]
```diff
@@ -90,6 +90,7 @@ export default defineComponent({
 
 The `@jsonforms/vue` package offers JSON Forms Core bindings based on the composition API.
 These bindings handle the props given to the `dispatch-renderer` and use the JSON Forms Core to determine specialized inputs for many use cases like validation and rule-based visibility.
+Using these bindings as a basis it's straightforward to create renderer sets for Vue 3.
```
Comment (lucas-koehler) [inline packages/vue/vue/README.md:93]:
```suggestion
Using these bindings as a basis, it's straightforward to create renderer sets for Vue 3.
```

File: packages/vue2/vue2/README.md [hunk 4]
```diff
@@ -99,6 +99,7 @@ Vue.use(VueCompositionAPI)
 
 The `@jsonforms/vue` package offers JSON Forms Core bindings based on the composition API.
 These bindings handle the props given to the `dispatch-renderer` and use the JSON Forms Core to determine specialized inputs for many use cases like validation and rule-based visibility.
+Using these bindings as a basis it's straightforward to create renderer sets for Vue 2.
```
Comment (lucas-koehler) [inline packages/vue2/vue2/README.md:102]:
```suggestion
Using these bindings as a basis, it's straightforward to create renderer sets for Vue 2.
```

**Review by lucas-koehler — COMMENTED** [review]
LGTM overall. I have some very minor remarks.

In the root README line 12, I would replace
> 2. Install dependencies with `npm ci` (or `npm install` when using an older version)

with:

> 2. Install dependencies with `npm ci`

As the JSONForms build does not work with node versions older than 14 (which includes npm 6), JSONForms cannot be built with an npm version not supporting the `ci` command

**Review by sdirix — APPROVED** [review]

**Pull request merged**